doc/src/selections.rst

```diff
@@ -44,48 +54,55 @@ false and false to true. ``name(#1) == H and not x(#1) < 5.0`` and ``(z(#2) < 45
 and name(#4) == O) or name(#1) == C`` are complex selections using booleans
 operators.
 
-In order to remove ambiguity when using multiple boolean operations, selections
-should use parentheses. If no parentheses are present, all operators will be
-treated as left-associatives.
-
-Selectors
----------
-
-The following selectors are implemented in chemfiles:
-
-- ``all``: select all the atoms;
-- ``none``: select none of the atoms;
-- ``type``: select atoms based on their type;
-- ``name``: select atoms based on their name. Some formats store both an atomic
-  name (H3) and an atom type (H), this is why you can use two different
-  selectors depending on the actual data;
-- ``index``: select atoms based on their index in the frame;
-- ``resname``: select atoms based on their residue name. An atom without residue
-  information will never be selected.
-- ``resid``: select atoms based on their residue name. An atom without residue
-  information will never be selected.
-- ``mass``: select atoms based on their mass;
-- ``x``, ``y`` and ``z``: select atoms based on their position cartesian components;
-- ``vx``, ``vy`` and ``vz``: select atoms based on their velocity cartesian components;
-
-Additional selectors ideas are welcome if they cannot be created easily from a
-combinaison of the above. If you don't know if your selector idea can be
-created from a combinaison, just ask!
+List of implemented properties
```

Comment:
I might open a separate issue for this - but maybe check the atom properties if the selector is unknown. For example, I could select all atoms with a property I've assigned specifically.

Reply:
Yes, this could be nice ! I would rather have a specific function for this:  `property("is_hetatm", #1)` or `prop("is_hetatm", #1)`; as property names might contain spaces.